### PR TITLE
[MBL-17266][Student][Teacher] Calendar render tests

### PR DESCRIPTION
**Pull request by tamaskozmer**
Test plan: Run tests

refs: MBL-17266
affects: Student, Teacher
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: libs/pandautils/src/androidTest/java/com/instructure/pandautils/compose/features/calendar/CalendarTest.kt [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+                calendarUiState = CalendarUiState(
+                    LocalDate.now(clock), true, stateMapper.createHeaderUiState(
+                        LocalDate.now(clock), null
+                    ), stateMapper.createBodyUiState(true, LocalDate.now(clock))
+                ),
+                actionHandler = {},
+            )
+        }
+
+        composeTestRule.onRoot().printToLog("TAG")
```
Comment (kristofnemere) [inline libs/pandautils/src/androidTest/java/com/instructure/pandautils/compose/features/calendar/CalendarTest.kt:71]: You left this here

**Review by kristofnemere — APPROVED** [review]

**Review by hermannakos — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**